### PR TITLE
Fixed redundant doi: text

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: templates/DatasetReview/submit.html.twig [hunk 1]
```diff
@@ -73,7 +73,7 @@
                     <b>DOI:</b>
                 </td>
                 <td>
-                    {{ DatasetSubmission.Dataset.Doi|default }}
+                    {{ DatasetSubmission.Dataset.Doi.getDoi|default }}
```
Comment (mickel1138) [inline templates/DatasetReview/submit.html.twig]: get is not needed. Twig automatically tries to find the function. get,has,is are some examples.

File: templates/Email/data-managers.dataset-submitted.email.twig [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
     Dear {{ recipient.firstName }} {{ recipient.lastName }},<br>
     <br>
-    A dataset has been submitted to the Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) by {{dataset.datasetSubmission.submitter.firstName}} {{dataset.datasetSubmission.submitter.lastName}} for the dataset {{dataset.udi}}. This dataset has been assigned the doi:{{dataset.doi|default}}. {{dataset.datasetSubmission.submitter.firstName}} {{dataset.datasetSubmission.submitter.lastName}} is a member of {{ submitterGroupsAndRepos | join (', ') }} and the dataset is associated with {{dataset.researchGroup.name}}.<br>
+    A dataset has been submitted to the Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) by {{dataset.datasetSubmission.submitter.firstName}} {{dataset.datasetSubmission.submitter.lastName}} for the dataset {{dataset.udi}}. This dataset has been assigned the doi:{{dataset.doi.doi|default}}. {{dataset.datasetSubmission.submitter.firstName}} {{dataset.datasetSubmission.submitter.lastName}} is a member of {{ submitterGroupsAndRepos | join (', ') }} and the dataset is associated with {{dataset.researchGroup.name}}.<br>
```
Comment (mickel1138) [inline templates/Email/data-managers.dataset-submitted.email.twig]: Here I would just remove the doi: text. And leave the property the way it was.

File: templates/DatasetReview/submit.html.twig (hunk 1 above)
Comment (praneethpr) [reply templates/DatasetReview/submit.html.twig]: Yes, I don't know why i put that get there . Will change it

File: templates/Email/user.dataset-created.email.twig [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <br>
     The Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) has received your dataset submission {{ datasetSubmission.dataset.udi }}.
     The dataset landing page is <a href="{{ url('pelagos_app_ui_dataland_default', { 'udi': datasetSubmission.dataset.udi}) }}">{{url('pelagos_app_ui_dataland_default', { 'udi': datasetSubmission.dataset.udi})}}</a>.<br>
-    This dataset has been assigned the doi: {{datasetSubmission.dataset.doi|default}} .<br><br>
+    This dataset has been assigned the doi: {{datasetSubmission.dataset.doi.doi|default}} .<br><br>
```
Comment (mickel1138) [inline templates/Email/user.dataset-created.email.twig]: Maybe we should make another function called getIdentifier that only suplies the identifier, with doi cut off?

**Comment by mickel1138** [comment]
There may a few more potential issues: `git grep -in dataset\.doi`

File: templates/Email/user.dataset-created.email.twig (hunk 3 above)
Comment (praneethpr) [reply templates/Email/user.dataset-created.email.twig]: getDoi() is already a function which does that

**Comment by praneethpr** [comment]
> There may a few more potential issues: `git grep -in dataset\.doi`

I found only 1 more, it was in tombstone

File: templates/Email/user.dataset-created.email.twig (hunk 3 above)
Comment (mickel1138) [reply templates/Email/user.dataset-created.email.twig]: I mean, one that is easier to read. Just a thought.

**Pull request merged**